src/common/components/repositories-list/index.js

```diff
@@ -23,9 +23,13 @@ const SNAP_NAME_NOT_REGISTERED_ERROR_CODE = 'snap-name-not-registered';
 export class RepositoriesListView extends Component {
 
   fetchAuthData(authStore) {
-    if (authStore.authenticated === null) {
+    // check if authenticated and hasShortNamespace haven't been set yet,
+    // so are null or undefined
+    if (authStore.authenticated === null ||
+        typeof authStore.authenticated === 'undefined') {
```

Comment:
Ah, it's a habit from old days when direct comparison to `undefined` as not safe (value of undefined could be changed by external script). But it seems it's not the case anymore with modern browsers and strict mode. Fixed.